go/grpcweb/trailer.go

```diff
@@ -0,0 +1,26 @@
+//Copyright 2018 Improbable. All Rights Reserved.
+// See LICENSE for licensing terms.
+
+package grpcweb
+
+import (
+	"net/http"
+	"strings"
+)
+
+// gRPC-Web spec says that must use lower-case header/trailer names.
+// See "HTTP wire protocols" section in
+// https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-WEB.md#protocol-differences-vs-grpc-over-http2
+type trailer struct {
+	http.Header
+}
+
+func (t trailer) Add(key, value string) {
```

Comment:
@johanbrandhorst should we not also override the `Get` method and automatically lower-case the supplied `key` parameter?

I would be worried that developers may get confused if they were to `Add()` a header with an upper-case key and then not be able to `Get()` it at a later date - maybe I'm over thinking.

Reply:
This is an unexported type, so it's not that important, and I don't think we use `trailer.Get` anywhere? I don't think it's necessary.